src/simularium/VisAgent.ts

```diff
@@ -414,42 +403,22 @@ export default class VisAgent {
 
         // set up new fiber as curved tube
         this.fiberCurve = new CatmullRomCurve3(curvePoints);
-        const fibergeometry = new TubeBufferGeometry(
-            this.fiberCurve,
-            4 * (numPoints - 1), // 4 segments per control point
-            collisionRadius * scale * 0.5,
-            8, // could reduce this with depth?
-            false
-        );
 
-        (this.mesh.children[0] as Mesh).geometry = fibergeometry;
-
-        if (this.mesh.children.length === 3) {
-            // update transform of endcap 0
-            const runtimeFiberEncapMesh0 = this.mesh.children[1] as Mesh;
-            runtimeFiberEncapMesh0.position.x = curvePoints[0].x;
-            runtimeFiberEncapMesh0.position.y = curvePoints[0].y;
-            runtimeFiberEncapMesh0.position.z = curvePoints[0].z;
-            runtimeFiberEncapMesh0.scale.x = collisionRadius * scale * 0.5;
-            runtimeFiberEncapMesh0.scale.y = collisionRadius * scale * 0.5;
-            runtimeFiberEncapMesh0.scale.z = collisionRadius * scale * 0.5;
-
-            // update transform of endcap 1
-            const runtimeFiberEncapMesh1 = this.mesh.children[2] as Mesh;
-            runtimeFiberEncapMesh1.position.x =
-                curvePoints[curvePoints.length - 1].x;
-            runtimeFiberEncapMesh1.position.y =
-                curvePoints[curvePoints.length - 1].y;
-            runtimeFiberEncapMesh1.position.z =
-                curvePoints[curvePoints.length - 1].z;
-            runtimeFiberEncapMesh1.scale.x = collisionRadius * scale * 0.5;
-            runtimeFiberEncapMesh1.scale.y = collisionRadius * scale * 0.5;
-            runtimeFiberEncapMesh1.scale.z = collisionRadius * scale * 0.5;
+        if (regenerateMesh) {
```

Comment:
I'm a little confused on where the fallback rendering vs non-fallback rendering are happening. This is where the fallback creation of TubeGeometries the slow way is being triggered right? I keep getting `GENERIC` vs `MOLECULAR` mixed up... I think some code comments about those would be helpful like @blairlyons said :)

Reply:
I changed the names GENERIC-->WEBGL1_FALLBACK and MOLECULAR-->WEBGL2_PREFERRED